File: Projects/server/Contracts/Responses/CarDto.cs
namespace server.Contracts.Responses
{
    public class CarDto
    {
        public uint CarId { get; set; }
        public string VinNumber { get; set; } = null!;
        public string? StateNumber { get; set; }
        public string BrandName { get; set; } = null!;
        public string ModelName { get; set; } = null!;
        public string BodyName { get; set; } = null!;
        public ushort ReleaseYear { get; set; }
        public string GearboxName { get; set; } = null!;
        public string DriveName { get; set; } = null!;
        public ushort VehicleWeightKg { get; set; }
        public ushort EnginePowerHp { get; set; }
        public float EnginePowerKw { get; set; }
        public float EngineCapacityL { get; set; }
        public byte TankCapacityL { get; set; }
        public string FuelTypeName { get; set; } = null!;
        public uint PersonId { get; set; }
        public uint PhotoId { get; set; }
    }
}

File: Projects/DataProviderApp/app/src/main/java/com/example/dataproviderapp/dto/responses/CarDto.kt
package com.example.dataproviderapp.dto.responses

data class CarDto(
    val carId: UInt,
    val vinNumber: String,
    val stateNumber: String?,
    val brandName: String,
    val modelName: String,
    val bodyName: String,
    val releaseYear: UShort,
    val gearboxName: String,
    val driveName: String,
    val vehicleWeightKg: UShort,
    val enginePowerHp: UShort,
    val enginePowerKw: Float,
    val engineCapacityL: Float,
    val tankCapacityL: UByte,
    val fuelTypeName: String,
    val personId: UInt,
    val photoId: UInt
)
